includes/derivatives.inc

```diff
@@ -126,9 +126,6 @@ function islandora_ocr_derive_datastreams(AbstractDatastream $datastream, array
   $ret = $ret && islandora_ocr_update_datastream($datastream->parent, $derivatives['OCR'], 'OCR');
   $ret = $ret && $derivatives['HOCR'];
   $ret = $ret && islandora_ocr_update_datastream($datastream->parent, $derivatives['HOCR'], 'HOCR');
-
-  $files = array_filter(array($source_file, $ocr_file, $hocr_file));
```

Comment:
Sorry - should have answered this sooner.  The temp files are passed to islandora_ocr_update_datastreams, and are dealt with [here](https://github.com/ajstanley/islandora_ocr/blob/7.x/includes/derivatives.inc#L465)